docs/src/cookbooks/two-side-column.md

```diff
@@ -0,0 +1,888 @@
+  output [31:0] add_res,
+  output [31:0] sub_res,
+  output [31:0] mod_res,
+  output [31:0] div_res,
+  output [31:0] and_res,
+  output [31:0] or_res,
+  output [31:0] xor_res,
+  output [31:0] not_res,
+  output [31:0] logical_not_res,
+  output [31:0] mux_res,
+  output [31:0] rshift_res,
+  output [31:0] lshift_res,
+  output        andR_res,
+  output        logical_and_res,
+  output        logical_or_res,
+  output        equ_res,
+  output        not_equ_res,
+  output        orR_res,
+  output        xorR_res,
+  output        gt_res,
+  output        lt_res,
+  output        geq_res,
+  output        leq_res,
+  output        single_bitselect_res,
+  output [63:0] mul_res,
+  output [63:0] cat_res,
+  output [1:0]  multiple_bitselect_res,
+  output [95:0] fill_res
+);
+  assign add_res = x + y; 
+  assign sub_res = x - y;
+  assign mod_res = x % y;
+  assign div_res = x / y;
+  assign and_res = x & y;
+  assign or_res = x | y; 
+  assign xor_res = x ^ y;
+  assign not_res = ~x; 
+  assign logical_not_res = !(x == 32'h0);
+  assign mux_res = c[0] ? x : y;
+  assign rshift_res = x >> y[2:0];
+  assign lshift_res = x << y[2:0];
+  assign logical_and_res = x[0] && y[0];
+  assign logical_or_res = x[0] || y[0];
+  assign equ_res = x == y; 
+  assign not_equ_res = x != y;
+  assign andR_res = &x;
+  assign orR_res = |x; 
+  assign xorR_res = ^x; 
+  assign gt_res = x > y; 
+  assign lt_res = x < y;
+  assign geq_res = x >= y; 
+  assign leq_res = x <= y;
+  assign single_bitselect_res = x[1];
+  assign mul_res = x * y;
+  assign cat_res = {x,y}; 
+  assign multiple_bitselect_res = x[1:0];
+  assign fill_res = {3{x}};
+endmodule
+
+```
+
+</td>
+<td>
+
+```scala mdoc:silent
+class OperatorExampleModule extends Module {
+
+  val x, y, c = IO(Input(UInt(32.W)))
+
+  val add_res, sub_res, mod_res, div_res, and_res, or_res, xor_res, not_res,logical_not_res, mux_res,  rshift_res , lshift_res = IO(Output(UInt(32.W)))
+  val logical_and_res, logical_or_res, equ_res, not_equ_res, andR_res, orR_res, xorR_res, gt_res,lt_res, geq_res, leq_res,single_bitselect_res = IO(Output(Bool()))
+  val mul_res, cat_res= IO(Output(UInt(64.W)))
+  val multiple_bitselect_res = IO(Output(UInt(2.W)))
+  val fill_res = IO(Output(UInt((3*32).W)))
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+  
+ 
+  add_res := x + y
+  sub_res := x - y 
+  mod_res := x % y
+  mul_res := x * y
+  div_res := x / y
+  equ_res := x === y
+  not_equ_res := x =/= y 
+  and_res := x & y
+  or_res := x | y
+  xor_res := x ^ y
+  not_res :=  ~x
+  logical_not_res := !x
+  logical_and_res := x(0) && y(0)
+  logical_or_res := x(0) || y(0)
+  cat_res := Cat(x, y)
+  mux_res := Mux(c(0), x, y)
+  rshift_res := x >> y(2, 0)
+  lshift_res := x << y(2, 0)
+  gt_res := x > y
+  lt_res := x < y
+  geq_res := x >= y
+  leq_res := x <= y
+  single_bitselect_res := x(1) 
+  multiple_bitselect_res := x(1, 0) 
+  fill_res:= Fill(3,x)
```

Comment:
can we get all these exactly the same order as the ones on the verilog so that they are side-by-side per operator?